src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1559,6 +1553,18 @@ public String toVariableNameString(EnchantmentOffer eo) {
 			.name("Experience Cooldown Change Reason")
 			.description("Represents a change reason of an <a href='events.html#experience cooldown change event'>experience cooldown change event</a>.")
 			.since("INSERT VERSION"));
+
+		if (Skript.classExists("org.bukkit.entity.EntitySnapshot")) {
+			Classes.registerClass(new ClassInfo<>(EntitySnapshot.class, "entitysnapshot")
+				.user("entity ?snapshots?")
+				.name("Entity Snapshot")
+				.description("Represents a snapshot of an entity's data. ",
+					"This includes all of the data associated with an entity (it's name, health, attributes, etc.), at the time it's retrieved. ",
+					"This acts as a template of sorts; it cannot be used to modify the entity which it represents.")
```

Comment:
```suggestion
					"Individual attributes of a snapshot cannot be modified or retrieved.")
```

Reply:
I think it sounded better before

Reply:
the template comment has been moved to the line above. i think this makes it more explicit that its only the template of the entity that you can copy